src/Components/PresentationalComponents/EmptyStates/EmptyStates.js

```diff
@@ -110,6 +110,19 @@ export const NotAuthorized = ({ title }) => (
     />
 );
 
+export const NoDescriptionAvailable = () => (
+    <Bullseye>
+        <EmptyState variant={EmptyStateVariant.large}>
```

Comment:
All empty states have `min-height` set to 250px by scss rule, you can make it not applicable here by editing the rule in `App.scss`

from:
`.pf-c-empty-state { ...`

to:
`.pf-c-empty-state:not(.no-description-available) { ...`

And adding `className="no-description-available"` to `EmptyState`, so now this rule does apply to all empty states that are not of class `no-description-available`

Also this EmptyState should not have padding, so:
`<EmptyState variant={EmptyStateVariant.large} style={{ padding: 0 }}>`

Also if you look at the mockup, the empty state's width is not restricted, but we have it restricted to 600px.